core/frontend/src/store/serviceHelper.ts

```diff
@@ -0,0 +1,27 @@
+import { Module, VuexModule, Mutation } from 'vuex-module-decorators'
+import store from '@/store'
+
+
+export interface Service {
+  valid: boolean;
+  title: string;
+  documentationUrl: string;
+  port: number;
+}
```

Comment:
I think it would be good for us to keep all the types/interfaces on `@/types/SERVICE.d.ts`.

Reply:
sounds good. so each one gets a file?

Reply:
Yup.
This way we keep the same separation for store, components and types.